### PR TITLE
Version command

### LINKED ISSUE #1165: Create command "version" to match style of other k8s binaries
## Feature Request

While creating a [bug report issue](https://github.com/operator-framework/operator-sdk/issues/1164) I tried to run the `operator-sdk version` command to find it does not exist as a command, but a flag `--version`

```10:06 $ operator-sdk version
Error: unknown command "version" for "operator-sdk"
Run 'operator-sdk --help' for usage.
10:06 $ operator-sdk --version
operator-sdk version v0.5.0+git
```
I would like to see `version` added as a command rather than as a flag to match the look and feel of other Kubernetes ecosystem CLIs like `kubectl`, etc.

### PR DESCRIPTION
**Pull request by Avni-Sharma**
Fixes #1165

### CODE REVIEW

File: commands/operator-sdk/cmd/version.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
+// Copyright 2018 The Operator-SDK Authors
```
Comment (joelanford) [inline commands/operator-sdk/cmd/version.go]:
```suggestion
// Copyright 2019 The Operator-SDK Authors
```

File: commands/operator-sdk/cmd/version.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+	ver "github.com/operator-framework/operator-sdk/version"
+
+	"github.com/spf13/cobra"
+)
+
+func NewVersionCmd() *cobra.Command {
+	versionCmd := &cobra.Command{
+		Use:   "version",
+		Short: "Provides version of the operator-sdk",
```
Comment (joelanford) [inline commands/operator-sdk/cmd/version.go]: Nit:
```suggestion
		Short: "Prints the version of operator-sdk",
```

**Comment by Avni-Sharma** [comment]
I am a bit confused with Travis CI here. Few moments back it showed all the tests to have been passed.
image

**Comment by LiliC** [comment]
@Avni-Sharma do you mind rebasing again, it seems your rebase includes one of my commits merged into master recently.

**Comment by hasbro17** [comment]
@Avni-Sharma Besides rebasing from master to remove the extra commits, you will also need to add a line to the `Added` section of the [CHANGELOG](https://github.com/operator-framework/operator-sdk/blob/master/CHANGELOG.md#added) and mention this change.

**Comment by Avni-Sharma** [comment]
Hi @hasbro17 . Made the suggested changes

**Comment by LiliC** [comment]
cc @hasbro17 if you want to have another look, otherwise we can merge this?

**Pull request merged**